recipes/python-pushover/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "python-pushover" %}
+{% set version = "0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: dee1b1344fb8a5874365fc9f886d9cbc7775536629999be54dfa60177cf80810
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
It looks like this has a python entry point. You should add it:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points

and then add a test under `commands`:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-commands